third_party/repo.bzl

```diff
@@ -16,6 +16,7 @@
 
 _SINGLE_URL_WHITELIST = depset([
     "arm_compiler",
+    "libxsmm_archive",
```

Comment:
Let's remove this line since we will have the mirror link.

Reply:
I will work on this next week, it's not forgotten. Let's take the time to finish it nicely.  I did not change the cluttered source code format so far to keep the patch small. I will happily pipe it through clang-format. Thank you for the review!